typescript/infra/src/utils/gcloud.ts

```diff
@@ -43,6 +44,29 @@ export async function fetchGCPSecret(
   return output;
 }
 
+export async function fetchLatestGCPSecret(secretName: string) {
+  const client = new SecretManagerServiceClient({
+    projectId: GCP_PROJECT_ID,
+  });
+  const [secretVersion] = await client.accessSecretVersion({
+    name: `projects/${GCP_PROJECT_ID}/secrets/${secretName}/versions/latest`,
+  });
+  const secretData = secretVersion.payload?.data;
+  if (!secretData) {
+    throw new Error(`Secret ${secretName} missing payload`);
+  }
+
+  // Handle both string and Uint8Array
```

Comment:
so this works for both?

```
secretValue
```

as well as 

```
{
 'x': 'secretValue',
 'y': 'secretOtherValue'
}
```

Reply:
the data type the SDK provides is a `string | Uint8Array` so this just converts to a string if we get a `Uint8Array`. Then if that string is a JSON or anything else it can be parsed later on